beetmoverscript/docker.d/worker.yml

```diff
@@ -131,6 +131,12 @@ bucket_config:
             key: { "$eval": "MAVEN_SNAPSHOT_KEY" }
           buckets:
             snapshot_components: 'maven-snapshots-s3-upload-bucket-knc0jfluv2kn'
+        maven-nightly-staging:
+          credentials:
+            id: { "$eval": "MAVEN_NIGHTLY_ID" }
+            key: { "$eval": "MAVEN_NIGHTLY_KEY" }
+          buckets:
+            nightly_components: 'maven-nightly-stage-s3-upload-bucket'
```

Comment:
Yes, as weird as this may seem, this is the bucket name. I double-checked too :)